### PR TITLE
Document GetFeatureInfo FILTER_GEOM

### LINKED ISSUE #1964: [FEATURE]: Select features in feature info intersecting a geometry
Original commit: https://github.com/qgis/QGIS/commit/e0efbda97fbb6ea50bad86832cde23d48388275a by mhugent

Unfortunately this naughty coder did not write a description... :-(

### PR DESCRIPTION
**Pull request by tudorbarascu**
fixes #1964

### CODE REVIEW

File: source/docs/user_manual/working_with_ogc/server/services.rst [hunk 1]
```diff
@@ -640,6 +640,20 @@ define the tolerance for point, line and polygon layers:
 * **FI_POLYGON_TOLERANCE** parameter: Tolerance for polygon layers
   *GetFeatureInfo* request, in pixels.
 
+QGIS Server also permits to make GetFeatureInfo requests and select all
+the features that intersect a geometry with the help of the **FILTER_GEOM**
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_ogc/server/services.rst]: maybe explain that it allow to user other geometry than Point. Standard GetFeatureInfo is only point in standard, so FILTER_GEOM extend this to polygon, right?

Comment (tudorbarascu) [reply source/docs/user_manual/working_with_ogc/server/services.rst]: @yjacolin Actually, it gets all the geometries that intersect a WKT geometry (not only a polygon)..

**Comment by tudorbarascu** [comment]
I reworded a little bit. Don't think we should go in super details.. of a GetFeatureInfo as the basics can be found everywhere and the WMS target users aren't super basic users, right?

**Comment by yjacolin** [comment]
I am ok with this!

**Comment by tudorbarascu** [comment]
@yjacolin Thanks for reviewing!

**Pull request merged**